tests/VersionRouteTest.php

```diff
@@ -289,4 +291,61 @@ public function testVersion(string $version, bool $isCorrectVersion, string $rou
 
         $response->assertStatus($status);
     }
+
+    public function testWithoutApiVersion(): void
+    {
+        $testCaseMock = (new TestCaseMock('name'))
+            ->setUpMock($this->app)
+            ->withoutAPIVersion();
+
+        Route::get('/test', function () {
+            return 'test';
+        });
+
+        $response = $testCaseMock->json('get', '/test');
+
+        $response->assertOk();
```

Comment:
let's configure mock class to assert that `call` method was called with expected uri